### DISCUSSION TITLE
Whitelabel

### DISCUSSION THREAD

**Discussion by egmose1224** [rustdesk/rustdesk#2219] [General]
Would it be possible to make an whitelabel solution, that still supports updates etc.? 

Maybe a config file, which has basic settings, like settings etc.

I know it´s a lot to ask, but i cross my fingers.

**Comment by mach7** [comment]
I'm interested in this as well.

**Comment by bzmrgonz** [comment]
Me 3, even if it's a paid service... premium offer.

**Comment by sebastianluebbert** [comment]
this would be great
